chatterbot/storage/django_storage.py

```diff
@@ -1,7 +1,5 @@
-import json
 from chatterbot.storage import StorageAdapter
 
-
```

Comment:
This line should not have been deleted. Python's style guide specifies that there should be two blank lines between imports and the a class definition. Once this is fixed I will merge this pull request.